portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Roller
+path: /universell-utforming/roller
+order: 7
+---
+
+# Universell utforming og roller
+Universell utforming er viktig igjennom hele organisasjonen, det er ikke en enkelt fagdisiplin eller gruppe som er alene ansvarlig. Det kan være en fare med kvalitetskrav som universell utforming som krever arbeid eller endring i arbeidet fra hele organisasjonen at det kan falle mellom to stoler eller at fagdisipliner dytter ansvaret mellom seg. For å unngå at det skjer er det viktig at alle har forståelse for hva som forventes av sin rolle og hva en skal forvente av andre.
+
+- [Alle](#alle)
+- [Designer](#designer)
+- [Utvikler](#utvikler)
+- [Tester](#tester)
+- [Produkteier/Teamlead](#produkteier-teamlead)
+
+## Alle
+Det forventes at alle har en grunnlegende forståelse for hva universell utforming er og hva det betyr for oss som organisasjon. Det forventes også at det er fokus på temaet, ikke fordi det er lovpålagt, men fordi det er det gjør hos Fremtind. Å bygge kvalitetskravene inn i prosessen og produktene våre tar tid, og det viktig at alle er med å drar i riktig retning og forstår at dette tar litt tid å få til riktig.
+
+## Designer
+Designere er "førsteforsvaret" for universell utforming. For enkelhets skyld er alle designdisiplinene slått sammen her, bør vurderes om det bør splittes på mer spesifikt nivå.
```

Comment:
@designers bør vi splitte forventningene for UX, visuelle designer, tjenestedesignere? Det er ikke noe jeg egentlig har noen formeninger om.